### ISSUE TITLE
Update Guide

### ISSUE THREAD

**Issue by andyduong1920** [nimblehq/compass-credo-plugin#30]
## Feature request

Should we update the guide, where the contributor know 

1. Where to start when they would like to introduce a new rule, like link to some Credo guide, blogs,...
2. How to apply and use this library in an existing Elixir project

**Comment by byhbt** [comment]
Agree, it's good to have the document for new contributors and one document for the usage of this compass plugin in the existing project
